tests/components/hassio/test_ingress.py

```diff
@@ -396,6 +396,37 @@ async def test_ingress_request_get_compressed(
     assert aioclient_mock.mock_calls[-1][3][X_FORWARDED_PROTO]
 
 
+@pytest.mark.parametrize(
+    "content_type",
+    [
+        "image/png",
+        "image/jpeg",
+        "font/woff2",
+        "video/mp4",
+    ],
+)
+async def test_ingress_request_not_compressed(
```

Comment:
Maybe also add a test for svg?

Reply:
79687646936d5cabc1e7a229aa15df98d3db5e0a